### ISSUE TITLE
Breaks my lovelace config?

### ISSUE THREAD

**Issue by jockebq** [TheLastProject/lovelace-valetudo-map-card#24]
Hi,

This is exactly what I am looking for, but I don't understand why it is not working.
I have put the files in `/www/community/lovelace-valetudo-map-card`
I have put the resource in the Raw config editor.
I have added the sensor in my configuration.yaml (should I change anything here?)

Then I added a "Manual card" in lovelace with the example you wrote in the readme. But after saving it shows a blank page on Home Assistant, everything in that tab is hidden. If I go out of editing mode I can see the other cards, but not the map, and when trying to remove the card it is not possible, as no cards are shown. I have to delete it with the Raw editor.

Thank you!

**Comment by TheLastProject** [comment]
What does your [browser console](https://kb.mailster.co/how-can-i-open-the-browsers-console/) say?

**Comment by jockebq** [comment]
@TheLastProject `TypeError: mapEntity.attributes.image is undefined
2 valetudo-map-card.js:257:22`

**Comment by TheLastProject** [comment]
Well, clearly your entity isn't of the correct type then and doesn't have the expected attributes.

I'll patch it later to at least not crash in this state, but you'll have to double-check the entity you gave the card.

**Comment by jockebq** [comment]
Where do I find the entity?
I can only find this: `vacuum.rockrobo`
I have installed Valetudo on my vacuum, I have connected it to MQTT which is also connected to Home Assistant, and I can control it.

By the way, is it possible to add this to HACS?

**Comment by TheLastProject** [comment]
Did you follow the steps in https://github.com/TheLastProject/lovelace-valetudo-map-card#configuration? You need to create a sensor in configuration.yaml.

The card also already is in HACS, it's named "Valetudo Map Card".

**Comment by jockebq** [comment]
Yes, I have created the sensor.
This is a part of my config.json on the Vacuum:
```
  "mqtt": {
    "enabled": true,
    "identifier": "rockrobo",
    "topicPrefix": "valetudo",
    "autoconfPrefix": "homeassistant",
    "broker_url": "mqtt://user:password@192.168.1.2",
    "provideMapData": true,
    "caPath": ""
  },
```


This is how I setup the sensor:
```
sensor:
  - platform: mqtt
    state_topic: "valetudo/roborock/state"
    json_attributes_topic: "valetudo/roborock/map_data"
    name: xiaomi_map
    value_template: 'OK'
    scan_interval: 5
```

The sensor shows up with this status in Home Assistant: 
`sensor.xiaomi_map  unknown  friendly_name: xiaomi_map`

**Comment by jockebq** [comment]
Solution was changing roborock to rockrobo. Don't know why it is this way as I am using stock settings on both. Now it works :)

**Comment by TheLastProject** [comment]
Ah, because the identifier in your config.json is rockrobo. I see now both Valetudo and Valetudo RE use rockrobo as example. I don't know why the example in this README uses roborock... Will update that.

**Comment by TheLastProject** [comment]
Fixed in ab051097c7a27bd53b43f6675cb500cf9aa02082 and 95d3aa3f04fa0ce939bba58af4a78936f4969033
